### DISCUSSION TITLE
aop里CurdBefore问题

### DISCUSSION THREAD

**Discussion by lsdlh** [dotnetcore/FreeSql#1444] [Q&A] [answered]
CurdBefore里DbParms取到的字段名字都转义了，怎么能取到真实字段名

**Comment by 2881099** [comment, marked as answer]
DbParms 是参数化数据。

如果想拦截 SetDto 可以考虑以下办法：

1、在代理层处理（比如自己定义的仓储或者DAL .Update 方法）

2、定义 SetDto2 方法，使用该方法替代 SetDto，并在该方法上处理

**Reply by lsdlh** [reply]
谢谢
